api/tests/core/finance/test_commands.py

```diff
@@ -113,3 +118,183 @@ def test_generate_invoices_internal_notification(app, css_font_http_request_mock
 
     assert call_kwargs["channel"] == "channel"
     assert call_kwargs["icon_emoji"] == ":large_green_circle:"
+
+
+@override_features(WIP_ENABLE_FINANCE_INCIDENT=True, WIP_ENABLE_NEW_BANK_DETAILS_JOURNEY=True)
+@clean_database
+def test_when_there_is_a_debit_note_to_generate_on_total_incident(app, css_font_http_request_mock):
+    sixteen_days_ago = datetime.datetime.utcnow() - datetime.timedelta(days=16)
+    fifteen_days_ago = datetime.datetime.utcnow() - datetime.timedelta(days=15)
+    fourteen_days_ago = datetime.datetime.utcnow() - datetime.timedelta(days=14)
+
+    user = users_factories.RichBeneficiaryFactory()
+    venue_kwargs = {
+        "pricing_point": "self",
+    }
+    offerer = offerers_factories.OffererFactory()
+    venue = offerers_factories.VenueFactory(managingOfferer=offerer, **venue_kwargs)
+    bank_account = finance_factories.BankAccountFactory(offerer=offerer)
+    bank_account_id = bank_account.id
+    offerers_factories.VenueBankAccountLinkFactory(
+        venue=venue, bankAccount=bank_account, timespan=[sixteen_days_ago, None]
+    )
+
+    # Invoice Part
+    book_offer = offers_factories.OfferFactory(venue=venue, subcategoryId=subcategories.LIVRE_PAPIER.id)
+    finance_factories.CustomReimbursementRuleFactory(amount=2850, offer=book_offer)
+
+    incident_booking1_event = finance_factories.UsedBookingFinanceEventFactory(
+        booking__stock=offers_factories.StockFactory(offer=book_offer, price=30, quantity=1),
+        booking__user=user,
+        booking__amount=30,
+        booking__quantity=1,
+        booking__dateCreated=sixteen_days_ago,
```

Comment:
La date de création de la réservation n'entre pas en ligne de compte. Pas besoin de la préciser. Par contre, on peut préciser ici la date d'utilisation : 

```suggestion
        booking__dateUsed=fifteen_days_ago,
```

... ce qui évite de bricoler le pricing et la réservation plus bas.